libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -236,9 +236,9 @@ AP_GPS_SBF::process_message(void)
 
         // Update position state (don't use −2·10^10)
         if (temp.Latitude > -200000) {
-            state.location.lat = (int32_t)(temp.Latitude * RAD_TO_DEG_DOUBLE * 1e7);
-            state.location.lng = (int32_t)(temp.Longitude * RAD_TO_DEG_DOUBLE * 1e7);
-            state.location.alt = (int32_t)(((float)temp.Height - temp.Undulation) * 1e2f );
+            state.location.lat = (int32_t)(temp.Latitude * RAD_TO_DEG_DOUBLE * (double)1e7);
+            state.location.lng = (int32_t)(temp.Longitude * RAD_TO_DEG_DOUBLE * (double)1e7);
+            state.location.alt = (int32_t)(((float)temp.Height - temp.Undulation) * 1e2f);
```

Comment:
not really obvious to me the need for double here. In the other lines there's the multiplication using RAD_TO_DEG_DOUBLE, here there isn't.